src/libraries/System.DirectoryServices/Directory.Build.props

```diff
@@ -5,6 +5,9 @@
          plan on shipping a new desktop version out of band. Instead add API
          to a different assembly. -->
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <!-- Since this assembly version is pinned, we don't want to validate that it matches
+    the expected assembly version on the targeting pack. -->
```

Comment:
Is this assembly actually in the targeting pack?  I wouldn't have expected it to be.

Reply:
I see, it is in the WindowsDesktop targeting pack.